.expeditor/scripts/post_habitat_release/shared.sh

```diff
@@ -31,3 +31,52 @@ worker_ident_for_target() {
     target="${1}"
     buildkite-agent meta-data get "${target}-builder-worker"
 }
+
+install_rustup() {
+  if command -v rustup && command -v cargo &>/dev/null; then
```

Comment:
@smacfarlane not a show stopper by any means, however, I'm curious what your thoughts are if we advertised these new functions up one directory as `./expeditor/scripts/shared.sh`? It just occurs to me that they might not be specific enough to be residing under `./expeditor/scripts/post_habitat_release/`.

Reply:
I went back on forth on that when I was copying the functions over from habitat-sh/habitat, but landed on this since they're not currently used anywhere else. I don't have really strong feelings either way though.

Reply:
Feels like an easy enough change to make should we want to. I'm :+1: on merging as is and coming back to juggle should we feel like we need to.

Reply:
Sure, let's move it later if needed.